scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -151,6 +151,13 @@ if [[ ${UPDATE_CHECK} = Y && ${SKIP_UPDATE} != Y ]]; then
     1) $0 "$@" "-u"; myExit 0 ;; # re-launch script with same args skipping update check
     2) exit 1 ;;
   esac
+
+  # check for CNCLI update
+  if [ -z "${CNCLI_STATUS}" ] ; then CNCLI_STATUS=$(check_cncli_versions) ; fi
```

Comment:
We dont really need to test if variable is previously set or not, it should always be overwritten on startup no matter what. So this could be changed to just:
```bash
CNCLI_STATUS=$(check_cncli_versions)
```

Reply:
That was old code so as to avoid the calling to repo API every 2 secs.